ui/components/ui/box/box.test.tsx

```diff
@@ -918,12 +917,6 @@ describe('Box', () => {
           <Box textAlign={TextAlign.Right}>Box right</Box>
           <Box textAlign={TextAlign.Justify}>Box justify</Box>
           <Box textAlign={TextAlign.End}>Box end</Box>
-
-          <Box textAlign={TEXT_ALIGN.LEFT}>DEPRECATED Box left</Box>
-          <Box textAlign={TEXT_ALIGN.CENTER}>DEPRECATED Box center</Box>
-          <Box textAlign={TEXT_ALIGN.RIGHT}>DEPRECATED Box right</Box>
-          <Box textAlign={TEXT_ALIGN.JUSTIFY}>DEPRECATED Box justify</Box>
-          <Box textAlign={TEXT_ALIGN.END}>DEPRECATED Box end</Box>
```

Comment:
Oh I was leaving these tests here to ensure the constant still works for all of our components maybe leave this till last?

Reply:
Ah yeah true, I guess you have all the clean up PRs ready to roll so it won't be a round for much longer. Fire away 🧹